recipes-kernel/linux/linux-mainline/openvario.cfg

```diff
@@ -105,19 +99,13 @@ CONFIG_SND_AUDIO_GRAPH_CARD=m
 # CONFIG_HID_REDRAGON is not set
 # CONFIG_HID_MICROSOFT is not set
 # CONFIG_HID_MONTEREY is not set
-CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
```

Comment:
I think this is a useful thing to keep enabled. Especially when someone complains their device doesn't work. We at least can undersdtand what kind of device is that from the log.

Reply:
The image contains `lsusb` which is much more useful, because prints more information. The kernel log is very brief.

Reply:
I often rely on the kernel logs to determine if device is working / recognized or not. If there is some blurp in the logs when I plug in the device, at least I know it is powered and talks to the host. Dunno, this feature helped my a lot in the past, I'd keep it.

Reply:
Changed so the option is kept.